plugins/auth/login.js

```diff
@@ -101,11 +102,13 @@ function addOAuthRoutes(server, config) {
                 );
                 request.cookieAuth.set(profile);
 
-                return factory.get({ username, scmContext })
+                return userFactory.get({ username, scmContext })
                     // get success, so user exists
                     .then((model) => {
                         if (!model) {
-                            return factory.create({
+                            // TODO: Move default collection creation here after database migration
```

Comment:
not sure if this is needed since the only time a default collection is really useful is when the user's pipelines are autopopulated in it

Reply:
I see the design calls for a default collection to exist no matter what